.github/workflows/docs.yml

```diff
@@ -18,13 +18,64 @@ name: "Documentation check"
 on:
 - pull_request
 
+env:
+  DEPS_BRANCH: bot/stable-deps-update
```

Comment:
# Question

Why not define this as an [environment variable of the job](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idenv) to keep things more tightly scoped?

Reply:
no reason, I'll try it out